src/components/WwwFrame/WwwPage.vue

```diff
@@ -70,11 +84,28 @@ export default {
 				fetchAllExpSettings(config, client, {
 					query: route?.query,
 					path: route?.path
-				})
+				}),
+				client.query({ query: optimizelyUserDataQuery })
```

Comment:
If there are no guards around this query, the data will be requested on every page load across the Ui application.

I think we need the check you previously had in place followed by wrapping the `readQuery` call in created in a try/catch statement. [Here's an example](https://github.com/kiva/ui/blob/main/src/pages/Lend/LoanChannelCategoryExperiment.vue#L875)